.github/workflows/release.yaml

```diff
@@ -29,7 +29,7 @@ jobs:
           fetch-depth: 0
       - name: Get changed files
         id: changed-markdown-files
-        uses: tj-actions/changed-files@3981e4f74104e7a4c67a835e1e5dd5d9eb0f0a57 # tag=v45.0.7
+        uses: tj-actions/changed-files@115870536a85eaf050e369291c7895748ff12aea # tag=v45.0.7
```

Comment:
I see v45.0.5 to v45.0.9 are all tagged to the same commit, and it's not this one, which is weird: https://github.com/tj-actions/changed-files/commit/a284dc1814e3fd07f2e34267fc8f81227ed29fb8. https://github.com/tj-actions/changed-files/commit/115870536a85eaf050e369291c7895748ff12aea is currently the tip of the main branch.

I wonder if they did some weird re-tagging thing after their recent security incident and ended up confusing dependabot. Maybe we should update this manually once.